### PR TITLE
Fix some typos in docs for `-C split-debuginfo`

### PR DESCRIPTION
**Pull request by camelid**

### CODE REVIEW

File: src/doc/rustc/src/codegen-options/index.md [hunk 1]
```diff
@@ -497,27 +497,27 @@ point instructions in software. It takes one of the following values:
...
   the final execution of `dsymutil` to generate debuginfo.
 
-* `packed` - This is the default for Windows MSVC and macOS platforms. The term
+* `packed` - This is the default for Windows MSVC and macOS. The term
   "packed" here means that all the debug information is packed into a separate
   file from the main executable. On Windows MSVC this is a `*.pdb` file, on
-  macOS this is a `*.dSYM` folder, and on other platforms this is a `*.dwp`
-  files.
+  macOS this is a `*.dSYM` folder, and on other platforms this is a collection
+  of `*.dwp` files.
```
Comment (camelid) [inline src/doc/rustc/src/codegen-options/index.md:512]: I couldn't tell when reading this is if it mean one `.dwp` file or a collection of files. I assumed it was the latter, but someone should probably check this.

Comment (Dylan-DPC) [reply src/doc/rustc/src/codegen-options/index.md:512]: the previous line in the paragraph states "in a single file" so i'm assuming it's just one file here.

Comment (steveklabnik) [reply src/doc/rustc/src/codegen-options/index.md:512]: so, the other changes look great to me, but this one needs to be changed, it seems?

File: src/doc/rustc/src/codegen-options/index.md [hunk 2]
```diff
@@ -497,27 +497,27 @@ point instructions in software. It takes one of the following values:
...
+  file.
 
 * `unpacked` - This means that debug information will be found in separate
   files for each compilation unit (object file). This is not supported on
   Windows MSVC. On macOS this means the original object files will contain
   debug information. On other Unix platforms this means that `*.dwo` files will
   contain debug information.
 
-Note that `packed` and `unpacked` gated behind `-Zunstable-options` on
+Note that `packed` and `unpacked` are gated behind `-Z unstable-options` on
```
Comment (camelid) [inline src/doc/rustc/src/codegen-options/index.md:520]: I added a space after `-Z` to make it easier to read. I think we use the with-space style elsewhere as well.

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
Looks great, thank you!

@bors: r+ rollup

**Pull request merged**